admin/src/lib.rs

```diff
@@ -74,15 +74,24 @@ pub async fn run(args: Args) {
             let certs = client.get_old_certificates().await.unwrap();
             eprintln!("Starting renewals of {} certs in 5 seconds...", certs.len());
             tokio::time::sleep(tokio::time::Duration::from_millis(5000)).await;
+            let mut any_error = false;
             for (cert_id, subject, acm) in certs {
                 println!(
-                    "--> {cert_id} {subject} {}",
+                    "{cert_id} {subject} {}",
                     if acm.is_some() { "(ACM)" } else { "" }
                 );
-                println!("{:?}", client.renew_certificate(&cert_id).await);
+                println!(
+                    "  {:?}",
+                    client.renew_certificate(&cert_id).await.inspect_err(|_| {
+                        any_error = true;
+                    })
+                );
                 // prevent api rate limiting
                 tokio::time::sleep(tokio::time::Duration::from_millis(1000)).await;
             }
+            if any_error {
+                panic!("Error in loop occured");
+            }
```

Comment:
Would this be cleaner in a utility function? Also, while we know what the error(s) was?